pynamodb/models.py

```diff
@@ -675,6 +675,33 @@ def scan(cls,
             rate_limit=rate_limit,
         )
 
+    @classmethod
+    def record_exists(cls, hash_key, range_key_condition=None, range_key=None):
```

Comment:
Maybe a way to avoid the "what if both arguments are passed" problem would be to have just one argument and branch on `isinstance(range_key, Condition)`?

Reply:
So this does match your current argument structure and naming where the second key is the condition expression (see `query`)

I am happy to do that branching mechanism but was concerned that it would be a divergent API compared to query and others (e.g. `MyModel.query('hash_key', 'range_key'` will not work today).

If you'd like me still reorder, rename, or remove something let me know!

Reply:
Yeah, it's `query(hash_key, range_key_condition=None)`. I guess we don't have a `query(hash_key, range_key)` shortcut form, but we want to have a shortcut form here?

Reply:
I'm happy to remove it if you'd rather not have it. In all my use cases I always want to check if a single record exists by looking up both the hash_key and range_key directly. The short form is nice since since i don't need to write out the whole range condition.

All that said, I'm happy to just drop the short form